### PR TITLE
update workload identity config

### PR DESCRIPTION
**Pull request by alvarorm22**
**Description of the change**

- [x] Update workload identity approach
- [x] Fix preflights when workload id is enabled

### CODE REVIEW

**Comment by alvarorm22** [comment]
https://app.shortcut.com/cartoteam/story/422123/update-workload-identity-config

File: manifests/kots-config.yaml [hunk 1]
```diff
@@ -802,11 +802,14 @@ spec:
...
-        - name: googleWorkloadIdentityEmail
-          title: Google workload identity email
+        - name: k8sWorkloadIdentityServiceAccountName
+          title: Kubernetes workload identity service account name
           when: '{{repl (ConfigOptionEquals "enableGoogleWorkloadIdentity" "1") }}'
           required: true
           type: text
+          help_text: |-
+            The kubernetes service account name you have created in the carto namespace.
+            Remember to annotate this SA with the GCP service account email.
```
Comment (moicalcob) [inline manifests/kots-config.yaml:812]: We'll have to add a link to the documentation and update the existing doc :)

Comment (alvarorm22) [reply manifests/kots-config.yaml:812]: yep, sure!

**Review by moicalcob — APPROVED** [review]
Come on! 🚀 🎸

**Pull request merged**